coco/state.py

```diff
@@ -364,7 +364,7 @@ def _load_default_state(self):
         for path, file in self.default_state_files.items():
             self.read_from_file(path, file)
 
-    async def process_post(self, request: dict):
+    async def process_post(self, request: dict = None):
         """
```

Comment:
I do not see this `request` variable referenced at any point in `process_post`. Is there a plan to do something with it later?

Reply:
This is to match the generic function signature from the caller `coco.endpoint.LocalEndpoint.call`. Is there a nicer way to ignore this unused parameter?

Reply:
Understood! This is fine if you want the signatures themselves to match. =)